exporter/opentelemetry-exporter-otlp-proto-http/pyproject.toml

```diff
@@ -28,7 +28,7 @@ dependencies = [
   "Deprecated >= 1.2.6",
   "backoff >= 1.10.0, < 2.0.0; python_version<'3.7'",
   "backoff >= 1.10.0, < 3.0.0; python_version>='3.7'",
-  "googleapis-common-protos ~= 1.52",
+  "googleapis-common-protos ~= 1.52, < 1.60.0",
```

Comment:
This is not loosening the requirement, this is making it more strict no? `googleapis-common-protos ~= 1.52` already allows any `1.x` version:

```sh
$ python3 -m venv venv 
$ ./venv/bin/pip install googleapis-common-protos ~= 1.52
$ ./venv/bin/pip list googleapis-common-protos
$ ./venv/bin/pip list
Package                  Version
------------------------ -------
googleapis-common-protos 1.59.1
pip                      23.0.1
protobuf                 4.23.3
setuptools               66.1.1
```

Reply:
Besides that, right now the [latest version of `googleapis-common-protos` is `1.59.1`](https://pypi.org/project/googleapis-common-protos/#history).

What is this change actually trying to prevent? Seems like you expect an issue to happen when `1.60.0` is released?